app/src/main/java/org/wikipedia/util/DateUtil.kt

```diff
@@ -0,0 +1,177 @@
+package org.wikipedia.util
+
+import android.icu.text.RelativeDateTimeFormatter
+import android.os.Build
+import android.text.format.DateFormat
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.feed.model.UtcDate
+import java.text.ParseException
+import java.text.SimpleDateFormat
+import java.util.*
+
+object DateUtil {
+    private val DATE_FORMATS: MutableMap<String, SimpleDateFormat> = HashMap()
```

Comment:
Can be simplified to:
`private val DATE_FORMATS = HashMap<String, SimpleDateFormat>()`